dist/src/main/resources/application.conf

```diff
@@ -39,7 +39,7 @@ sparkta {
     sparkHome = ""
     deployMode = cluster
     numExecutors = 2
-    masterDispatchers = 127.0.0.1
+    master =  "mesos://127.0.0.1:5050"
```

Comment:
If you mean double quotes, yes, they're needed because of the colon.
```
ConfigException$Parse: reference.conf: 50: Expecting close brace } or a comma, got ':' (if you intended ':' to be part of the value for 'master', try enclosing the value in double quotes, or you may be able to rename the file .properties rather than .conf)
```